interactive/templates/interactive/analysis_request_output.html

```diff
@@ -7,57 +7,108 @@
   <header class="bg-oxford-50">
+
+    {% if user.is_staff %}
+
+    <div class="container py-8 sm:pt-16 pb-14 grid md:grid-cols-2 items-start gap-4">
+      <div class="flex flex-col-reverse">
+        <h1 id="analysisHeading" class="mt-2 text-4xl font-bold text-slate-900 sm:text-5xl sm:tracking-tight lg:text-6xl">
+          {{ analysis.codelist_name }}
+        </h1>
+        <p class="font-bold text-oxford-600 tracking-wide uppercase">
+          {% if analysis.complete_email_sent_at %}
+            {{ analysis.complete_email_sent_at|date:"d M Y" }}
+          {% else %}
+            Analysis Output
+          {% endif %}
+        </p>
+      </div>
+      <div class="rounded-md bg-white shadow p-4">
+        <div class="prose prose-oxford prose-sm text-slate-800">
+          <h2 class="border-b pb-2 mb-2">Staff actions</h2>
+          <ul>
+            <li>
+              <strong>User:</strong>
+              {{ analysis.user.name }} ({{ analysis.user.email }})
+            </li>
+            <li>
+              <strong>Commit:</strong>
+              <a class="no-underline hover:text-oxford-800 hover:underline focus:text-oxford-800 focus:underline" href="https://github.com/opensafely/interactive/commit/{{ analysis.commit_sha }}">
+                {{ analysis.commit_sha }}
+              </a>
+            </li>
+            <li>
+              <strong>Email sent to user:</strong>
+              {{ analysis.complete_email_sent_at|yesno:"Yes,No" }}
+              {% if analysis.complete_email_sent_at %}
+                &mdash; {{ analysis.complete_email_sent_at | naturaltime }}
+              {% endif %}
+              <ul>
+                <li>
+                  {% url 'request_analysis_email' analysis.id as email_url %}
+                  {% if analysis.complete_email_sent_at %}
+                    {% include "components/button.html" with el="link" html_class="hover:text-oxford-800 hover:underline focus:text-oxford-800 focus:underline" text="Resend email to requester" href=email_url %}
+                  {% else %}
+                    {% include "components/button.html" with el="link" html_class="hover:text-oxford-800 hover:underline focus:text-oxford-800 focus:underline" text="Send an email to requester" html_class="hover:oxford-800 hover:underline" href=email_url %}
+                  {% endif %}
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+
+    {% else %}
+
     <div class="container py-8 sm:pt-16 pb-14">
       <div class="text-center">
         <div class="flex flex-col-reverse">
           <h1 id="analysisHeading" class="mt-2 text-4xl font-bold text-slate-900 sm:text-5xl sm:tracking-tight lg:text-6xl">
-            {{ analysis.title }}
+            {{ analysis.codelist_name }}
           </h1>
           <p class="font-bold text-oxford-600 tracking-wide uppercase">
-            Analysis Output
+            {% if analysis.complete_email_sent_at %}
```

Comment:
Is this something we want to show to most users?

Reply:
I was leaving this here so that users could have some way of distinguishing two reports with the same codelist/title from each other. I believe there is a ticket to add a "created at" date to the model, which we could use here instead.

Reply:
That's a good ideea. It is confusing using the complete_email_sent_at field, so I think moving to the `created at` field soon would make sense (probably not in this PR though).

For reference, the `created_at` field now exists in main. However, I doubt it's been back-filled, so will always be None at the moment.